dpe/src/commands/sign.rs

```diff
@@ -198,6 +202,15 @@ pub struct SignP384Cmd {
     pub digest: [u8; 48],
 }
 
+#[repr(C)]
+#[derive(Debug, PartialEq, Eq, IntoBytes, FromBytes, Immutable, KnownLayout)]
+pub struct SignMldsaExternalMu87Cmd {
+    pub handle: ContextHandle,
+    pub label: [u8; 48],
+    pub flags: SignFlags,
+    pub digest: [u8; 48],
```

Comment:
This should be 64 bytes.

But also, related to 1 v.s. 2 profiles, let's chat about whether it makes sense to make the to-be-signed data hash-sized or 1k.